capstone/src/main/java/com/google/step/data/RestaurantPageViews.java

```diff
@@ -33,6 +35,12 @@ public String getName() {
         return pageViews;
     }
 
+    /**
+     * Gets the id of the instance
+     * @return String representing the id of the restaurant.
+     */
+    public String getId() { return id; }
```

Comment:
Nit: it doesn't matter that much, but consider changing this to be consistent with the other getters.

Reply:
Done.